quint/test/docs.test.ts

```diff
@@ -0,0 +1,32 @@
+import { assert } from 'chai'
+import { describe, it } from 'mocha'
+import { produceDocs, toMarkdown } from '../src/docs'
+import { buildModuleWithDefs } from './builders/ir'
+
+describe('produceDocs', () => {
+  const module = buildModuleWithDefs([
+    `/// This is a docstring for foo\nval foo = 1`,
```

Comment:
Could this and similar be multi line strings? I think it would help with readability.